### DISCUSSION TITLE
Sudden serialized closure error, difficult to fix

### DISCUSSION THREAD

**Discussion by JF-626164** [laravel/framework#48579] [Q&A]
Hello all,

This morning my application started encountering this issue when attempting to upload files:
`Your serialized closure might have been modified or it's unsafe to be unserialized. {"exception":"[object] (Laravel\\SerializableClosure\\Exceptions\\InvalidSignatureException(code: 0): Your serialized closure might have been modified or it's unsafe to be unserialized. at /../../../vendor/laravel/serializable-closure/src/Serializers/Signed.php:83)`

In the past when this issue appeared, I would regenerate the keys and then clear the route, cache, config, and view, then the problem would be fixed.

Any idea how to fix this? I've gone through all the recent changes, and I don't see anything that could've caused this.

_Edit note:_ This problem only occurs on my dev server; my local development environment is not encountering this issue.

**Comment by henzeb** [comment]
How is this serialization been done? Queue?

**Reply by JF-626164** [reply]
Yes, a Bus::chain() to be exact. I run a number of jobs when this particular import occurs.

**Comment by JF-626164** [comment]
4 days later the issue is gone. Not certain as to why just yet.

**Reply by tamayoac** [reply]
what is the fix here? I am also having a issues similar with yours.

**Reply by JF-626164** [reply]
I don't know what the fix is. I ran composer update and php artisan optimize:clear a ridiculous amount of times and eventually it went away.

**Comment by iamdarzee** [comment]
composer require laravel/serializable-closure 
  fixed it for me
